terraform/context_apply_test.go

```diff
@@ -7461,10 +7461,9 @@ func TestContext2Apply_targetedDestroy(t *testing.T) {
 		t.Fatalf("expected 0 resources, got: %#v", mod.Resources)
 	}
 
-	// the root output should have been removed too, since it is derived solely
-	// from the targeted resource
-	if len(mod.OutputValues) != 0 {
-		t.Fatalf("expected 0 outputs, got: %#v", mod.OutputValues)
+	// the root output should not get removed
```

Comment:
Maybe change this comment to indicate that we're verifying the current status-quo, but it's not necessarily incorrect to remove the output if we can manage it in the future.